sn_networking/src/lib.rs

```diff
@@ -403,6 +404,7 @@ impl SwarmDriver {
             // 63 will mean at least 63 most recent peers we have dialed, which should be allow for enough time for the
             // `identify` protocol to kick in and get them in the routing table.
             dialed_peers: CircularVec::new(63),
+            unroutable_peers: CircularVec::new(127),
```

Comment:
why 127 here vs 63 above?

Reply:
The `UnroutablePeer` event occurs regularly and is not bad news per se. My reasoning is that this vector is filled quite quickly, therefore we need a bit more memory.